spec/requests/admin/reports_spec.rb

```diff
@@ -19,7 +19,7 @@
 
     it 'has a link to the download csv path' do
       subject
-      expect(response.body).to include(admin_reports_submitted_csv_path)
+      expect(response.body).to include(admin_reports_submitted_csv_path(format: :csv))
```

Comment:
Again, what are we changing that we suddenly need to send the explicit format 😕

Reply:
because previously, the repsonse body contained `/admin/admin_report_submitted.csv` and we were seeing if it included  `admin_reports_submitted_csv_path`, which was `/admin/admin_report_submitted`.

Now, the response_body contains `/admin/admin_report_submitted.csv?locale=en` and the `admin_reports_submitted_csv_path` without the format param equates to `/admin/admin_report_submitted?locale=en`, hence the need to add the format param to get this test to pass